docs/upgradeFromD8ToD9.md

```diff
@@ -36,22 +35,22 @@ docker ps
 
 ### Step 3:
 
-Now let's tar.gz the whole ensemble with data and configs. As an example we will save this into your `$HOME` folder. 
-As a good practice we append the **current date **(YEAR-DAY-MONTH) to the filename. Here we assume today is December 1st of 2021.
+Now let's `tar.gz` the whole ensemble with data and configs. As an example we will save this into your `$HOME` folder. 
+As a good practice we append the **current date **(YEAR-MONTH-DAY) to the filename. Here we assume today is December 1st of 2021.
 
 ```Shell
 cd ..
-sudo tar -czvpf $HOME/archipelago-deployment-RC2-20210112.tar.gz archipelago-deployment
+sudo tar -czvpf $HOME/archipelago-deployment-RC2-20211201.tar.gz archipelago-deployment
```

Comment:
I do not want to have the date formar conversation but ok, we can roll with this change